app/src/main/index.js

```diff
@@ -0,0 +1,73 @@
+'use strict'
+
+import { app, BrowserWindow } from 'electron'
+import * as path from 'path'
+import { format as formatUrl } from 'url'
+
+const is_development = process.env.NODE_ENV !== 'production'
+
+let main_window_keep_alive
+
+function create_main_window() {
+    const window = new BrowserWindow(
+        {
+            width          : 1024,
+            height         : 768,
+            frame          : false,
+        }
+    )
+
+    var search = "";
+    if (app.commandLine.hasSwitch("debug")) {
+        var value = app.commandLine.getSwitchValue("debug");
```

Comment:
First you are checking if something exist, then yo uare getting it, then you are checking if its undefined. This is low quality code.